code/model/DNEnvironment.php

```diff
@@ -1160,6 +1160,11 @@ public function onAfterDelete() {
 		if($this->config()->get('allow_web_editing') && $this->envFileExists()) {
 			unlink($this->getConfigFilename());
 		}
+
+		$create = $this->CreateEnvironment();
```

Comment:
yeah... we'll probably refactor the whole create env thing later anyway. Good to merge?